src/transformers/models/whisper/generation_whisper.py

```diff
@@ -1131,33 +1135,52 @@ def replace_or_add(lst: List[int], num: int, itr: Iterator[int]):
                     f"You are using token ids in `forced_decoder_ids` that do not seem to correctly follow the prompt pattern of Whisper. Make sure that {forced_decoder_ids} has an entry for all indices >= 1 and < {forced_decoder_ids[0][0]}.",
                 )
 
-        # from v4.39 the forced decoder ids are always None in favour of decoder input ids
-        generation_config.forced_decoder_ids = None
```

Comment:
Is this an intentional deletion? Note that we're moving away from `forced_decoder_ids` in favour of the `language`/`task` arguments